new-lamassu-admin/src/pages/Maintenance/CashboxHistory.js

```diff
@@ -111,21 +110,27 @@ const CashboxHistory = ({ machines, currency }) => {
   }
 
   const save = row => {
+    const _performedBy = R.prop(
+      'performedBy',
+      R.find(f => f.id === row.id, fields)
+    )
+
+    const performedBy = _performedBy === '' ? null : _performedBy
+
     schema
-      .isValid(fields)
+      .isValid(performedBy)
```

Comment:
Looking at the `schema`, I don't understand why this is working... From my understanding, the variable given to `isValid()` should be an object with the key `"performedBy"`.
But again, it's working on release too, so maybe it's my misunderstanding.